CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Airship Xamarin Changelog
 
+## Version 14.0.0 - Sept 28, 2020
+Major release to support iOS SDK 14.1.2 & Android SDK 14.0.1.
+
+### Changes
```

Comment:
We also added some methods on deleting/reading messages and fixes the channel update listener.

Reply:
@oristanovic has docs that need https://urbanairship.atlassian.net/browse/MOBILE-1924 in as well. Looks like a small lift, can we get it in real fast for the release?